packages/webview_flutter/lib/src/webview_method_channel.dart

```diff
@@ -40,6 +40,8 @@ class MethodChannelWebViewPlatform implements WebViewPlatformController {
       case 'onPageFinished':
         _platformCallbacksHandler.onPageFinished(call.arguments['url']);
         return null;
+      case 'onProgress':
+        _platformCallbacksHandler.onProgress(call.arguments['progress']);
```

Comment:
The last merge lost the `return null;`.

```suggestion
        _platformCallbacksHandler.onProgress(call.arguments['progress']);
        return null;
```

Reply:
Thanks, it's fixed 👍